stream_alert/alert_processor/outputs.py

```diff
@@ -407,14 +413,22 @@ def dispatch(self, **kwargs):
         if not creds:
             return self._log_status(False)
 
+        # Cache base_url
+        self._base_url = creds['api']
+
         # Preparing headers for API calls
         self._headers = {
             'Authorization': 'Token token={}'.format(creds['token']),
             'Accept': 'application/vnd.pagerduty+json;version=2'
         }
 
-        # Cache base_url
-        self._base_url = creds['api']
+        # Get user email to be added as From header and verify
+        user_email = creds['email_from']
+        if not self._user_verify(user_email, False):
+            return self._log_status(False)
+
+        # Add From to the headers after verifying
+        self._headers['From'] = user_email
```

Comment:
is this header value really title case (`From`)?

Reply:
It seems so: https://v2.developer.pagerduty.com/docs/incident-creation-api

Reply:
weird 🤣